fastlane/Jetpack-Fastfile

```diff
@@ -145,7 +145,7 @@ lane :update_jetpack_appstore_strings do |options|
     'screenshot-text-4' => File.join(source_metadata_folder, 'promo_screenshot_4.txt'),
     'screenshot-text-5' => File.join(source_metadata_folder, 'promo_screenshot_5.txt'),
     'screenshot-text-6' => File.join(source_metadata_folder, 'promo_screenshot_6.txt'),
-    'app-store-name' => File.join(source_metadata_folder, 'app_name.txt')
+    app_store_name: File.join(source_metadata_folder, 'app_name.txt')
```

Comment:
I updated this key used in the `Jetpack-Fastfile` to be the same as the WordPress one. It's safe to do this update because the value was _never used_ since it wasn't originally in the Jetpack `.po` file. This can be verified by noticing there's no `app-store-name` on the Jetpack GlotPress page. E.g.:

![image](https://user-images.githubusercontent.com/1218433/157783073-4752e883-627d-44a3-8cee-a94e1307467c.png)